packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -412,6 +413,118 @@ Content-Type: application/json;charset=UTF-8
 }
 ```
 
+### /bc/authorize
+
+<ApiOperation method="post" url="${baseUrl}/v1/bc/authorize" /> <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />
+
+This endpoint returns a unique identifier (`auth_request_id`) that identifies the authentication flow while it tries to authenticate the user in the background.
+
+>  **Note:** This endpoint's base URL varies depending on whether you are using a Custom Authorization Server. For more information, see [Composing your base URL](#composing-your-base-url).
+
+#### Request parameters
+
+You can post the following parameters as a part of the URL-encoded form values to the API.
+
+> **Note:** The `/bc/authorize` endpoint requires client authentication. See the [Client authentication methods](#client-authentication-methods) section for more information on which method to choose and how to use the parameters in your request.
+
+| Parameter               | Description                                                                                                                                                                                     | DataType | Required |
+| :---------------------- | :----------------------------------------------------------------------------------------------------------------------------                                                                   | :-----   | :-----   |
+| scope                   | `openid` is required for authentication requests. You can also include other [scopes](#access-token-scopes-and-claims).                                                                         | String   | TRUE     |
+| login_hint              | A hint to the OpenID Provider regarding the user for whom authentication is being requested.                                                                                                    | String   | TRUE     |
+| id_token_hint           | An ID token previously issued to the client as a hint to identify the user for whom authentication is being requested.                                                                          | String   | TRUE     |
+| binding_message         | A message that appears for the user to identify the transaction.                                                                                                                                | String   | FALSE    |
+| request_expiry          | A positive integer allowing the client to request the `expires_in` value for the `auth_req_id` that the server returns. Max value is five minutes.                                              | Integer  | FALSE    |
+| request                 | A JWT created by the client that enables requests to be passed as a single, self-contained parameter. See [Parameter details](#client-initiated-backchannel-authentication-parameter-details).  | JWT      | FALSE    |
```

Comment:
is this optional?
```suggestion
| request                 | (Optional) A JWT created by the client that enables requests to be passed as a single, self-contained parameter. See [Parameter details](#client-initiated-backchannel-authentication-parameter-details).  | JWT      | FALSE    |
```

Reply:
![image](https://user-images.githubusercontent.com/79472517/196818994-be7070bc-0d92-4bc2-b10c-7ce8d81fbf2a.png)
yes